src/operators/elementAt.ts

```diff
@@ -0,0 +1,58 @@
+import Operator from '../Operator';
+import Observer from '../Observer';
+import Subscriber from '../Subscriber';
+import ArgumentOutOfRangeError from '../util/ArgumentOutOfRangeError';
+
+export default function elementAt(idx: number, defaultValue?: any) {
+  return this.lift(new ElementAtOperator(idx, defaultValue));
+}
+
+class ElementAtOperator<T, R> implements Operator<T,R> {
+  
+  idx: number;
+  defaultValue: any;
+  
+  constructor(idx: number, defaultValue?: any) {
+    if (idx < 0) {
+      throw new ArgumentOutOfRangeError;
+    }
+    
+    this.idx = idx;
+    this.defaultValue = defaultValue;
+  }
+  
+  call(subscriber: Subscriber<T>): Subscriber<T> {
+    return new ElementAtSubscriber(subscriber, this.idx, this.defaultValue);
+  }
+}
+
+class ElementAtSubscriber<T, R> extends Subscriber<T> {
+  
+  idx: number;
```

Comment:
will do. Just curiosity - does guideline recommends to init with default value when it's declared? i.e, 
`private idx:number = 0` vs `private idx:number`. Personally do not have any specific preferences.